addons/pos_sale/models/sale_order.py

```diff
@@ -45,12 +45,6 @@ def _compute_qty_delivered(self):
         for sale_line in self:
             sale_line.qty_delivered += sum([self._convert_qty(sale_line, pos_line.qty, 'p2s') for pos_line in sale_line.pos_order_line_ids if sale_line.product_id.type != 'service'], 0)
 
-    @api.depends('pos_order_line_ids.qty')
```

Comment:
This is not required anymore as the sale invoiced_qty is correctly computed, because the invoice line are linked to the sale_order_line